apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -136,7 +136,16 @@ class PageTemplateModal extends Component {
 		// Only overwrite the page title if the template is not one of the Homepage Layouts
 		const title = isHomepageTemplate ? null : this.getTitleByTemplateSlug( slug );
 
-		// Skip inserting if there's nothing to insert.
+		// Check to see if this is a blank template selection
+		// and reset the template if so.
+		if ( 'blank' === slug ) {
```

Comment:
Not a blocker, but this can be run before [the `isHomepageTitle` and  `getBlocksByTemplateSlug()` bits](https://github.com/Automattic/wp-calypso/pull/39204/files#diff-a664d83ace51cb87304bf656c9e9f303R132) to return earlier, since we're not using those variables.

Reply:
Good catch, I can move this, then I'll merge. 👍